nvrdftd-taipei.md

```diff
@@ -0,0 +1,10 @@
+# Taipei, Taiwan
+
+### Restaurants for Eaters
+
+### Attractions for Tourists
+
+- Taipei 101
+- National Palace Museum
```

Comment:
@githubteacher The south or east is typically a good option for your visit. Most of the time people make reservations way before the vacation starts, so it will be definitely sure to get ahead. 👏